tests/runtime/watchpoint

```diff
@@ -1,7 +1,9 @@
 NAME watchpoint - absolute address
-RUN {{BPFTRACE}} -e 'watchpoint:0x10000000:8:w { printf("hit!\n"); exit() }' -c ./testprogs/watchpoint
+BEFORE ./testprogs/watchpoint
+RUN {{BPFTRACE}} -e "watchpoint:$(awk '{print $1}' /tmp/watchpoint_mem):8:w { printf(\"hit!\n\"); exit() }" -p {{BEFORE_PID}}
 EXPECT hit!
 ARCH aarch64|ppc64|ppc64le|x86_64
+CLEANUP /bin/bash -c "rm -f /tmp/watchpoint_mem"
```

Comment:
I think CLEANUP already runs in a shell, no? So I think you can drop the `/bin/bash -c ...` bit